tools/ansible/tasks/wsu/main.yaml

```diff
@@ -151,3 +162,23 @@
       until: '"k8s.ovn.org/hybrid-overlay-distributed-router-gateway-mac" in overlay.stdout'
       retries: 12
       delay: 5
+
+    - name: Get hybrid overlay subnet
+      delegate_to: localhost
+      shell: oc get nodes "{{ node_name.stdout }}" -o=jsonpath='{.metadata.annotations.k8s\.ovn\.org/hybrid-overlay-hostsubnet}'
+      register: hybrid_subnet
+
+    - name: Create cni config
+      win_copy:
+        content: "{\"cniVersion\":\"0.2.0\",\"name\":\"OpenShiftNetwork\",\"type\":\"win-overlay\",\"capabilities\":{\"dns\":true},\"ipam\":{\"type\":\"host-local\",\"subnet\":\"{{ hybrid_subnet.stdout }}\"},\"policies\":[{\"name\":\"Endpoin
+tPolicy\",\"value\":{\"Type\":\"OutBoundNAT\",\"ExceptionList\":[\"10.132.0.0/16\"]}},{\"name\":\"EndpointPolicy\",\"value\":{\"Type\":\"ROUTE\",\"DestinationPrefix\":\"10.132.0.0/16\",\"NeedEncap\":true}}]}"
+        dest: "{{ win_temp_dir.path }}\\cni.conf"
```

Comment:
There will be a double copy of `cni.conf` now. One will be present in `c:\k\cni\` and another in `c:\k\cni\config`. Please put this in `{{ win_temp_dir.path }}\\cni\config\cni.conf`...